ew.py

```diff
@@ -665,8 +665,8 @@ def get_vouchers(self):
 
 	def get_festivity(self):
 		data = ewutils.execute_sql_query(
-		"SELECT {festivity} + COALESCE(sigillaria, 0) + FLOOR({coin_gambled} / 1000000000000) FROM users "\
-		"LEFT JOIN (SELECT {id_user}, {id_server}, COUNT(*) * 100 as sigillaria FROM items INNER JOIN items_prop ON items.{id_item} = items_prop.{id_item} "\
+		"SELECT FLOOR({festivity}) + COALESCE(sigillaria, 0) + FLOOR({coin_gambled} / 1000000000000) FROM users "\
```

Comment:
festivity is already stored as an int in the database

Reply:
I changed it to a float so that it can properly store festivity gains from slimecoin.

Reply:
Ah sorry, I should have updated it in the readme to make it more clear.

Reply:
was there a problem with storing coin gambled separately the way we have been?

Reply:
Yes, Kanman (and Trackpad too I believe) already hit the cap on bigint.

Reply:
To be more specific, Kanman can no longer gain festivity from gambling slimecoin, as he has reached the max possible value slimernalia_coin_gambled can store